girder/api/v1/system.py

```diff
@@ -156,8 +157,27 @@ def getVersion(self, params):
         .errorResponse('You are not a system administrator.', 403)
     )
     def enablePlugins(self, plugins, params):
+        pluginWebroots = plugin_utilities.getPluginWebroots()
+        setting = self.model('setting')
+        routeTable = setting.get(SettingKey.ROUTE_TABLE)
+        oldPlugins = setting.get(SettingKey.PLUGINS_ENABLED)
+        reservedRoutes = (GIRDER_ROUTE_ID, GIRDER_STATIC_ROUTE_ID)
```

Comment:
If we just make `reservedRoutes = {GIRDER_ROUTE_ID, GIRDER_STATIC_ROUTE_ID}`, we won't have to copy it into a set further down.